core/backend/src/ECSchemaOps.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module ECSchemaOps
+ */
+
+import { IModelHost } from "./IModelHost";
+
+/**
+ * Generate SHA1 Hash of Schema XML
+ * @param schemaXmlPath: Path where schema XML file is located
+ * @param referencePaths: Schema reference paths
+ * @param isExactMatch: Schema references are located by exact scheme version comparisons
+ * @public
+ */
+export function generateSchemaSha1Hash(schemaXmlPath: string, referencePaths: string[], isExactMatch: boolean = false): string {
```

Comment:
The signature to this function should be:
```ts
export function generateSchemaSha1Hash(arg: {schemaXmlPath: string, referencePaths: string[], isExactMatch?: boolean): string {
```
And i still think that should be the signature of the native method too (rather than having 2 methods.)

I don't understand why you need a try/catch here either. Why isn't the native exception sufficient?

Loose methods are discouraged, and I suggest you follow Paul's suggestion and either add `namespace Schema` here, or simply make this (now one line) method on `IModelHost` or `BisCoreSchema` (since every schema *is* a BisCoreSchema that's entirely logical IMHO) instead of adding a new file.

Reply:
I made a [draft PR](https://bentleycs.visualstudio.com/iModelTechnologies/_git/imodel02/pullrequest/255373) for you that makes a single native method with named arguments. Please combine that PR with this one. 

BTW, you named the native method "compute", this method should likewise be "compute" rather than "generate".

Reply:
Agreed! I will do these modifications.